### PR TITLE
Added some explanatory notes for the SUMD serial protocol

### PR DESCRIPTION
**Pull request by mha1**
changes:
- added a paragraph with some information om SUMD to the serial protocols section
- added an example on how to setup a FBL using the SUMD protocol

Please let me know if wording and formatting is ok.

### CODE REVIEW

File: docs/software/serial-protocols.md [hunk 1]
```diff
@@ -57,3 +57,31 @@ When connected to the receiver with the transmitter, enter the ExpressLRS Lua sc
 <figure markdown>
 <img class="center-img" src="../../assets/screenshots/receiver-serial.png" width="auto">
 </figure>
+
+## SUMD Notes
+
+SUMD is the Graupner HoTT bus protocol to transmit multiple channel data over a single serial line. It transports up to 16 full res channels at a rate of 100Hz and is mainly used to provide digital channel data to Flybarless Control Systems for helicopters and older Flight Controllers.
+
+As already stated many times in this manual ExpressLRS expects AUX1 (channel 5) to control the arm state. This may lead to a conflict using serial protocols like SUMD in conjunction with device having fixed channel assignments. To circumvent the problem SUMD implements a channel remapping by swapping channels 5 and 8 in its serial output:
```
Comment (deadbytefpv) [inline docs/software/serial-protocols.md:65]: "times in this manual, ExpressLRS"
comma

"conjunction with a device" or "conjunction with devices"

"To circumvent the problem, SUMD"
comma

File: docs/software/serial-protocols.md [hunk 2]
```diff
@@ -57,3 +57,31 @@ When connected to the receiver with the transmitter, enter the ExpressLRS Lua sc
...
+
+<figure markdown>
+<img class="center-img" src="../../assets/images/sumd1.png" width="auto">
+</figure>
+
+This opens the possibility to assign a linear output channel order for channels 1 to 7 using an appropriate OpenTX/EdgeTX mixer setup.
+
+Example: Microbeast flybarless controller without RPM governor
+
+Follow the Microbeast manual to setup the single line receiver mode.
```
Comment (deadbytefpv) [inline docs/software/serial-protocols.md:75]: Example: Microbeast flybarless controller without RPM governor. (Follow the Microbeast manual to setup the single line receiver mode.)

**Review by deadbytefpv — CHANGES_REQUESTED** [review]
Just a few minor changes.

File: docs/software/serial-protocols.md [hunk 3]
```diff
@@ -57,3 +57,31 @@ When connected to the receiver with the transmitter, enter the ExpressLRS Lua sc
 <figure markdown>
 <img class="center-img" src="../../assets/screenshots/receiver-serial.png" width="auto">
 </figure>
+
+## SUMD Notes
+
+SUMD is the Graupner HoTT bus protocol to transmit multiple channel data over a single serial line. It transports up to 16 full res channels at a rate of 100Hz and is mainly used to provide digital channel data to Flybarless Control Systems for helicopters and older Flight Controllers.
```
[the remark this answers is not in the document]
Comment (mha1) [reply docs/software/serial-protocols.md]: done

File: docs/software/serial-protocols.md (hunk 1 above)
Comment (mha1) [reply docs/software/serial-protocols.md]: done

File: docs/software/serial-protocols.md (hunk 2 above)
Comment (mha1) [reply docs/software/serial-protocols.md]: done

**Comment by mha1** [comment]
@deadbytefpv thanks for your review

**Review by deadbytefpv — APPROVED** [review]

**Pull request merged**